package.json

```diff
@@ -25,7 +25,7 @@
     "test:watch": "jest --config jest.test.config.js --watch",
     "test:visual": "jest --config jest.visual.config.js --runInBand",
     "run-if:percy": "node scripts/run-percy.js",
-    "percy": "percy exec -- jest --config jest.visual.config.js --runInBand",
+    "vrt": "percy exec -- jest --config jest.visual.config.js --runInBand",
```

Comment:
What would speak against keeping some "command consistency" across UIKIt and app-kit?

Reply:
I wanted to inspect the percy CLI command by doing `yarn run percy --help` and instead this command was executed 🙃

<img width="510" alt="image" src="https://user-images.githubusercontent.com/1110551/53091768-2bfa8580-3513-11e9-9980-8ae083efd911.png">

Reply:
Ah okay. Nice catch. Maybe we can change the command at UIKit then too to avoid this? 👍